frontend/app/forms/motion-to-waive-fees/motion-to-waive-fees.web.component.js

```diff
@@ -1,23 +1,580 @@
+          <GroupSelect
+            dataKey="case.courtAddress"
+            label="Justice Courts"
+            groupOptions={JusticeCourtList}
+          />
+        )}
+
+        <TextInput dataKey="case.caseNumber" label="Case number" />
+        <TextInput dataKey="case.judgeName" label="Judge" />
+        <TextInput dataKey="case.commissionerName" label="Commissioner" />
+        <small className="web-form-input">(Domestic cases)</small>
+      </Section>
+
+      <Section name="3. Fees to waive">
+        <p className="web-form-input">
+          I ask for the following fee(s) to be waived:
+        </p>
+        <small className="web-form-input">(Choose all that apply)</small>
+        <CheckBox
+          dataKey="case.filingFee"
+          label="Filing fee"
+          name="feesToWaive"
+        />
+        <small className="web-form-input">(Refer to Civil Cover Sheet)</small>
+        {data.case.filingFee && (
+          <TextInput label="Filing Fee Amount" dataKey="case.filingFeeAmount" />
+        )}
+        <CheckBox dataKey="case.ocapFee" label="OCAP fee" name="feesToWaive" />
+        <small className="web-form-input">($20.00)</small>
+
+        <CheckBox
+          name="feesToWaive"
+          dataKey="case.divorceEducation"
+          label="Divorce education class fee"
+        />
+        <small className="web-form-input">(Maximum $35.00)</small>
+
+        <CheckBox
+          dataKey="case.divorceOrientation"
+          label="Divorce orientation class fee"
+          name="feesToWaive"
+        />
+        <small className="web-form-input">(Maximum $30.00)</small>
+
+        <CheckBox
+          dataKey="case.vitalRecordFee"
+          label="Office of Vital Records fee"
+          name="feesToWaive"
+        />
+        <small className="web-form-input">
+          (Certification of Adoption or Certificate of Divorce; $8.00)
+        </small>
+
+        <CheckBox
+          dataKey="case.serviceFee"
+          label="Service Fee"
+          name="feesToWaive"
+        />
+        <small className="web-form-input">(within Utah)</small>
+
+        <CheckBox
+          dataKey="case.appealFee"
+          label="Appeal fee"
+          name="feesToWaive"
+        />
+        {data.case.appealFee && (
+          <>
+            <CheckBox
+              label="$225 Filing fee"
+              dataKey="case.appealFeeLarge"
+              name="filingFeeAmount"
+            />
+            <CheckBox
+              label="$10 small claims appeal"
+              dataKey="case.appealFeeSmall"
+              name="filingFeeAmount"
+            />
+            <small className="web-form-input">(Justice Court)</small>
+          </>
+        )}
+        <CheckBox
+          dataKey="case.otherFeeToWaive"
+          label="Other"
+          name="feesToWaive"
+        />
+        {data.case.otherFeeToWaive && (
+          <>
+            <TextInput label="Explain" dataKey="case.otherFeeToWaive.name" />
```

Comment:
I think there's a problem here with the data keys.

`case.otherFeeToWaive.name` and `case.otherFeeToWaive`.

The `.` in the data key means "object". But we're trying to store a checkbox's boolean value into `otherFeeToWaive`. And then later we're thinking that `otherFeeToWaive` is an object.

This should be fixed. It matters for both in-memory data storage and for when the user logs in and the data is stored to firebase.